src/Admin.php

```diff
@@ -71,12 +71,12 @@ public static function init() {
    * @see http://www.paulund.co.uk/hide-wordpress-update-notice
    * @see http://premium.wpmudev.org/blog/hide-the-wordpress-update-notification/
    */
-  public static function removeUselessCoreUpdateNagMessages() {
-    if (!current_user_can('update_core')) {
+  public static function removeNagNotices() {
       // @see wp-admin/includes/admin-filters.php
       remove_action('admin_notices', 'update_nag', 3);
       remove_action('admin_notices', 'maintenance_nag', 10);
-    }
+      //removes all nag notices including from woocommerce
```

Comment:
All code comments (as well as commit messages) should form proper English sentences, starting with an capitalized letter and ending with punctuation.  Have a look at the rest of the file to see many good examples.

We also have (or had) these guidelines written down somewhere, which would be a better read than getting one by one feedback about individual coding style issues in your PRs, but I'm not able to find locate them right now (see Slack).